modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -1051,6 +1051,8 @@ def astype(self, col_dtypes, **kwargs):
         HdkOnNativeDataframe
             The new frame.
         """
+        if not isinstance(col_dtypes, dict):
```

Comment:
```suggestion
        if isinstance(col_dtypes, str):
```
Should we explicitly check for str?